packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -2156,6 +2156,9 @@ export class ContainerRuntime
 		const modernRuntimeMessage = messageArg.type === MessageType.Operation;
 
 		// Do shallow copy of message, as the processing flow will modify it.
+		// There might be multiple container instances receiving same message
+		// We do not need to make deep copy, as each layer will just replace message.content itself,
```

Comment:
```suggestion
		// We do not need to make a deep copy. Each layer will just replace message.content itself,
```

Reply:
Side note - Honestly this comment makes me nervous but for this PR I'm just copying it as-is (your edits are fine)